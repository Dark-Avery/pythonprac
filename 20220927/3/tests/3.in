123,234,345
456,567,678
789,8910,101112
10,10,10
10,20,30
40,50,60