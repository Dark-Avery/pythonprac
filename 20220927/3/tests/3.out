17370,23160,28950
37350,49800,62250
4141470,5241690,6341910
